### PR TITLE
organizaion slug

### PR DESCRIPTION
**Pull request by felire**
## Summary

Organization slug for bitrise configuration

### CODE REVIEW

File: generators/app/tasks/createBitriseApp.js [hunk 1]
```diff
@@ -4,90 +4,85 @@ const gitlabApi = require('../apis/gitlabApiConfig');
...
-        git_owner: gitOwner,
-        is_public: isPublic,
-        provider: provider,
-        repo_url: repoUrl,
-        git_repo_slug: repoSlug,
-        type: type
-      })
-};
+const createApp = async ({ repoUrl, isPublic, gitOwner, provider, repoSlug, type }) =>
+  await bitriseApi.post('/apps/register', {
```
Comment (sfernandez11) [inline generators/app/tasks/createBitriseApp.js]: is this `await` necessary?

File: generators/app/templates/__tests__/screens/Login/testLoginScreen.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import CustomTextInput from '@components/CustomTextInput';
+import { shallow } from 'enzyme';
```
Comment (sfernandez11) [inline generators/app/templates/__tests__/screens/Login/testLoginScreen.js]: Global imports first! Move this line above `import CustomTextInput from '@components/CustomTextInput';`

File: generators/app/tasks/createBitriseApp.js [hunk 3]
```diff
@@ -4,90 +4,85 @@ const gitlabApi = require('../apis/gitlabApiConfig');
...
-        })
+const createSshGit = ({ repoSlug, publicSshKey, provider }) => {
+  // IMPORTANT -> GITLAB AND GITHUB HAVE THE SAME EP
+  if (provider === 'bitbucket') {
+    return createSshOnBitbucket(repoSlug, publicSshKey);
+  }
+  return gitApi[provider].post('/user/keys', {
+    title: `bitrise-${repoSlug}`,
+    key: publicSshKey
+  });
```
Comment (rodrigoWolox) [inline generators/app/tasks/createBitriseApp.js]: You could do 
```
const createSshGit = ({ repoSlug, publicSshKey, provider }) =>
  provider === 'bitbucket'
    ? createSshOnBitbucket(repoSlug, publicSshKey)
    : gitApi[provider].post('/user/keys', {
        title: `bitrise-${repoSlug}`,
        key: publicSshKey
      });
```

File: generators/app/tasks/createBitriseApp.js [hunk 4]
```diff
@@ -4,90 +4,84 @@ const gitlabApi = require('../apis/gitlabApiConfig');
...
-};
+const createApp = ({ repoUrl, isPublic, gitOwner, provider, repoSlug, type }) =>
+  bitriseApi.post('/apps/register', {
+    git_owner: gitOwner,
+    is_public: isPublic,
+    provider,
+    repo_url: repoUrl,
+    git_repo_slug: repoSlug,
+    type,
+    organization_slug: '91af5dfae6f01368'
```
Comment (WFolini) [inline generators/app/tasks/createBitriseApp.js]: Be careful, this repo is public and is thought for use not only Wolox. All Bitrise data will be configurable via params or env vars.

**Pull request merged**